### ISSUE TITLE
Support for multiple organisations in chronograf

### ISSUE THREAD

**Issue by ajayanaj** [regel/loudml#135]
Hi,

We are looking to use loudml in our system, but while testing we noticed that organisations administration is missing in your customised chronograf. We are using chronograf for multiple customers (each influx database for each customers) with multiple organisation. 
Say example I have a customer1 and the DB looks like customer1_db and the chronograf org looks like customer1. This will continue as customer2,3 etc.
Is there any plan to add this feature with chronograf? so that we offer ML possibilities to our customers.
Also, we noticed that the chronograf version is old (1.6), we are using 1.7.11 to get flux support.
It would be great if you can use the latest version of chronograf.

**Comment by regel** [comment]
@ajayanaj thanks for trying Loud ML. Can you please send an email to contact [at] loudml.io with your contact and company information?

**Comment by regel** [comment]
@ajayanaj if I understand correctly you are requesting a TICK-L upgrade to 1.7.x plus the addition of organizations. Is this correct? What about the integration in Grafana?

**Comment by robcowart** [comment]
+1 for Grafana!

**Comment by activiteOCR** [comment]
+1 for Grafana!

**Comment by ajayanaj** [comment]
@regel  Yes, that's correct. I would like to see if this can be integrated with Grafana.
